subsys/nrf_security/src/drivers/CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ else()
 	add_subdirectory(nrf_cc3xx)
   endif()
 
-  if(CONFIG_PSA_CRYPTO_DRIVER_CRACEN)
+  if(CONFIG_PSA_CRYPTO_DRIVER_CRACEN OR (CONFIG_NRF_SECURITY_LEGACY_AND_PSA AND CONFIG_SOC_SERIES_NRF54LX))
```

Comment:
Seems a bit counterintuitive as in the new case `CONFIG_PSA_CRYPTO_DRIVER_CRACEN` is not selected, but still added in the CMake logic.

Reply:
Yes, it is indeed counterintuitive but for this particular use case necessary. For the  legacy and PSA config we need to include that to get RNG even though we don't use the rest of the driver. We use Oberon for everything except RNG. This is an experimental feature (legacy and PSA) which we hope to get it removed later when then adoption of PSA is better.